src/components/admin/PortalStripForm.tsx

```diff
@@ -195,22 +193,22 @@ export const PortalStripForm: React.FC<PortalStripFormProps> = ({
           </div>
         </div>
         <div className="PortalStripForm__cell PortalStripForm__visibility">
-          <AdminCheckbox
+          <Checkbox
             variant="toggler"
             name="isClickable"
             register={register}
             label={renderedClickableLabel}
-            labelPosition="after"
+            // labelPosition="after"
```

Comment:
Like the above comment about using `//`, would be nicer to either delete or provide an explanation in code. Alternatively something like a disable flag as noted in https://github.com/sparkletown/sparkle/blob/redesign/world-settings/docs/code-style.md#maybe-could-be-useful-later-dead-code

Reply:
The above comment is valid for all similar commented out props, wouldn't want to spam it everywhere.

Reply:
agree. If needed, can be added in the future.
removed this kind of comment: https://github.com/sparkletown/sparkle/pull/2874/commits/9f96015f2ded1d8da0274c00a2e1323adf454d70